### PR TITLE
Update AWS support message

### PR DESCRIPTION
**Pull request by ArielLima**
Updating AWS support message reflecting vCPU usage as a limit for regions.

### CODE REVIEW

**Comment by clcollins** [comment]
On lines 1048-1049, `caseLimitIncreaseBody` is appended to `caseCommunicationBody`, but is no longer necessary.

Also, on line 1053, there is duplicate language in the appended message: "**Once this is completed please resolve this case, do not set this case to Pending Customer Action.**"

File: pkg/controller/account/account_controller.go [hunk 1]
```diff
@@ -1022,19 +1022,8 @@ func (r *ReconcileAccount) statusUpdate(reqLogger logr.Logger, account *awsv1alp
...
-	instanceTypeList := strings.Join(coveredInstanceTypes, ", ")
-
-	// For each supported AWS region append to the communication a request of limit increase
-	var i = 0
-	for region := range coveredRegions {
-		i++
-		caseLimitIncreaseBody := fmt.Sprintf("Limit increase request %d\nService: EC2 Instances\nRegion: %s\nPrimary Instance Type: %s\nLimit name: Instance Limit\nNew limit value: %d\n------------\n", i, region, instanceTypeList, caseDesiredInstanceLimit)
-		caseCommunicationBody += caseLimitIncreaseBody
-	}
+	caseCommunicationBody := fmt.Sprintf("Hello AWS, Please enable Enterprise Support on AWS account %s\n\nOnce this has been completed and the default EC2 limits are ready for use, please resolve this support case.\n\nThanks", accountID)
```
Comment (jharrington22) [inline pkg/controller/account/account_controller.go]: Missing new lines after "Hello AWS"

File: pkg/controller/account/account_controller.go [hunk 2]
```diff
@@ -1022,19 +1022,8 @@ func (r *ReconcileAccount) statusUpdate(reqLogger logr.Logger, account *awsv1alp
...
+	caseSubject := fmt.Sprintf("Add account %s to Enterprise Support", accountID)
 
 	// Per AWS suggestion add final msg to case body
 	caseCommunicationBody += "\n\n**Once this is completed please resolve this case, do not set this case to Pending Customer Action.**"
```
Comment (jharrington22) [inline pkg/controller/account/account_controller.go]: This can be removed now?

Comment (ArielLima) [reply pkg/controller/account/account_controller.go]: That's what I understood from @clcollins

Comment (jharrington22) [reply pkg/controller/account/account_controller.go]: You haven't removed it though?

Comment (ArielLima) [reply pkg/controller/account/account_controller.go]: Done

File: pkg/controller/account/account_controller.go (hunk 1 above)
Comment (ArielLima) [reply pkg/controller/account/account_controller.go]: Done

File: pkg/controller/account/account_controller.go [hunk 3]
```diff
@@ -1022,22 +1022,8 @@ func (r *ReconcileAccount) statusUpdate(reqLogger logr.Logger, account *awsv1alp
...
-	var i = 0
-	for region := range coveredRegions {
-		i++
-		caseLimitIncreaseBody := fmt.Sprintf("Limit increase request %d\nService: EC2 Instances\nRegion: %s\nPrimary Instance Type: %s\nLimit name: Instance Limit\nNew limit value: %d\n------------\n", i, region, instanceTypeList, caseDesiredInstanceLimit)
-		caseCommunicationBody += caseLimitIncreaseBody
-	}
-
-	// Per AWS suggestion add final msg to case body
-	caseCommunicationBody += "\n\n**Once this is completed please resolve this case, do not set this case to Pending Customer Action.**"
+	caseCommunicationBody := fmt.Sprintf("Hello AWS,\n\nPlease enable Enterprise Support on AWS account %s\n\nOnce this has been completed and the default EC2 limits are ready for use, please resolve this support case.\n\nThanks", accountID)
```
Comment (clcollins) [inline pkg/controller/account/account_controller.go]: I think this lost the bit about the "Pending Customer Action" stuff.

Maybe just:

`...Once this has been completed and the default EC2 limits are ready for use, please resolve this support case.  Please do not set the case to Pending Customer Action....`

Comment (ArielLima) [reply pkg/controller/account/account_controller.go]: done

**Pull request merged**